CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Optimized state synchronizations by removing unnecessary fetching and parsing of note details (#462).
 - [BREAKING] Changed `GetAccountDetailsResponse` field to `details` (#481).
 - Improve `--version` by adding build metadata (#495).
+- [BREAKING] Introduced additional limits for note/asset/transaction number (#503). 
```

Comment:
The note and transaction limits are enforced in the batch builder, but where is the `asset` limit checked?